src/implementations/twig/components/media-container/media-container.story.js

```diff
@@ -1,71 +1,115 @@
-import he from 'he';
-import { withKnobs, text, select, object } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl/storybook-addon-notes';
 import withCode from '@ecl/storybook-addon-code';
-import { getExtraKnobs, tabLabels } from '@ecl/story-utils';
 
-// Import data for demos
-import demoImg from '@ecl/specs-component-media-container/demo/data--image';
-import demoVideo from '@ecl/specs-component-media-container/demo/data--video';
-import demoEmbed from '@ecl/specs-component-media-container/demo/data--embed-video';
+import dataImg from '@ecl/specs-component-media-container/demo/data--image';
+import dataVideo from '@ecl/specs-component-media-container/demo/data--video';
+import dataEmbed from '@ecl/specs-component-media-container/demo/data--embed-video';
 import mediaContainer from './media-container.html.twig';
 import notes from './README.md';
 
-const prepareMediaContainer = (data, media) => {
+const getArgTypes = (data, media) => {
+  const argTypes = {};
+
   if (media === 'video') {
```

Comment:
I don't think you need to create a new media param here. >
Is this condition not enough ?

```
if (data.embedded_media) {
...
} else if (data.sources) {
...
}
```

Reply:
I kept the same logic as in the original file, but I'll check if it can be simplified